### PR TITLE
Upgrade third party deps

### PR DESCRIPTION
**Pull request by borkaehw**
1. Fixed a typo in the `README`.
2. Fixed buildifier never works.
3. Upgrade third party deps.

### CODE REVIEW

File: scripts/format.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ else
 fi
 
 if [ "$1" != check ]; then
-    bazel build buildifier
+    bazel run buildifier
```
Comment (borkaehw) [inline scripts/format.sh]: It should be `run` not `build`. It never works.

File: WORKSPACE [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
+skydoc_sha256 = "8762a212cff5f81505a1632630edcfe9adce381479a50a03c968bd2fc217972d"
 
 http_archive(
     name = "io_bazel_skydoc",
-    sha256 = "694602137e5d56cfd35622cf58c047549324a0db1522ee944ad86e74420be9db",
-    strip_prefix = "skydoc-b8a32e07ee8297c89ca8020af4fa2163a766706f",
-    urls = ["https://github.com/bazelbuild/skydoc/archive/b8a32e07ee8297c89ca8020af4fa2163a766706f.zip"],
+    sha256 = skydoc_sha256,
+    strip_prefix = "skydoc-{}".format(skydoc_tag),
+    urls = ["https://github.com/bazelbuild/skydoc/archive/{}.zip".format(skydoc_tag)],
```
Comment (SrodriguezO) [inline WORKSPACE]: can we use `url` instead?

File: WORKSPACE [hunk 3]
```diff
@@ -1,29 +1,29 @@
...
+buildtools_sha256 = "05eb52437fb250c7591dd6cbcfd1f9b5b61d85d6b20f04b041e0830dd1ab39b3"
+
 http_archive(
     name = "com_github_bazelbuild_buildtools",
-    sha256 = "998ccc97f78c4ee959ad7ebb5effed30a6a8afc7891e6be3c2e0355900846c7d",
-    strip_prefix = "buildtools-19db42aa7a206a52b67bc66477c069ca83d092f4",
-    urls = ["https://github.com/bazelbuild/buildtools/archive/19db42aa7a206a52b67bc66477c069ca83d092f4.zip"],
+    sha256 = buildtools_sha256,
+    strip_prefix = "buildtools-{}".format(buildtools_tag),
+    urls = ["https://github.com/bazelbuild/buildtools/archive/{}.zip".format(buildtools_tag)],
```
Comment (SrodriguezO) [inline WORKSPACE]: Same here

File: scripts/format.sh (hunk 1 above)
Comment (SrodriguezO) [reply scripts/format.sh]: good catch

File: WORKSPACE (hunk 3 above)
Comment (borkaehw) [reply WORKSPACE]: Fixed.

File: WORKSPACE (hunk 2 above)
Comment (borkaehw) [reply WORKSPACE]: Fixed.

**Pull request merged**